### ISSUE TITLE
Example fix?

### ISSUE THREAD

**Issue by dotemacs** [ajith-ab/react-native-receive-sharing-intent#4]
The example doesn't work as is. The error it comes back with is:

```
$ pod install
Detected React Native module pod for react-native-receive-sharing-intent
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Flipper-Folly":
  In snapshot (Podfile.lock):
    Flipper-Folly (= 2.2.0, ~> 2.1, ~> 2.2)

  In Podfile:
    Flipper-Folly (~> 2.1)

None of your spec sources contain a spec satisfying the dependencies: `Flipper-Folly (~> 2.1), Flipper-Folly (= 2.2.0, ~> 2.1, ~> 2.2)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
```

To get it to work, I had to remove `Podfile.lock` and re-run `pod install` which resolved the issue.

The diff is:

```diff
modified   example/ios/Podfile.lock
@@ -233,7 +233,7 @@ PODS:
     - React-cxxreact (= 0.62.2)
     - React-jsi (= 0.62.2)
   - React-jsinspector (0.62.2)
-  - react-native-receive-sharing-intent (1.0.0):
+  - react-native-receive-sharing-intent (1.0.2):
     - React
   - React-RCTActionSheet (0.62.2):
     - React-Core/RCTActionSheetHeaders (= 0.62.2)
@@ -349,7 +349,7 @@ DEPENDENCIES:
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://github.com/cocoapods/specs.git:
+  trunk:
     - boost-for-react-native
     - CocoaAsyncSocket
     - CocoaLibEvent
@@ -443,7 +443,7 @@ SPEC CHECKSUMS:
   React-jsi: b6dc94a6a12ff98e8877287a0b7620d365201161
   React-jsiexecutor: 1540d1c01bb493ae3124ed83351b1b6a155db7da
   React-jsinspector: 512e560d0e985d0e8c479a54a4e5c147a9c83493
-  react-native-receive-sharing-intent: b49e04fd81d1da416c0e8af37cd0bb1f040de3ab
+  react-native-receive-sharing-intent: 78f444ac1c1ae8b2652a844de1820d5009c44d16
   React-RCTActionSheet: f41ea8a811aac770e0cc6e0ad6b270c644ea8b7c
   React-RCTAnimation: 49ab98b1c1ff4445148b72a3d61554138565bad0
   React-RCTBlob: a332773f0ebc413a0ce85942a55b064471587a71
@@ -459,4 +459,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 56c2537f71f3f02200d6918c542a8e89a0b422fa
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.9.1
```

I can do this as a separate PR or just add it to: #3 ?

**Comment by dotemacs** [comment]
Maybe we can wait for you to merge #3 and then I can update the Podfile.lock which will pull in the next version: `1.0.4`?

But if we're updating `Podfile.lock` can we also commit in `package-lock.json` so that the dependencies are in sync for the example?

If we you don't want to update `Podfile.lock` and commit `package-lock.json`, then can we remove `Podfile.lock`?

**Comment by ajith-ab** [comment]
@dotemacs I updated in repo

Thanks 
Ajith A B

**Comment by dotemacs** [comment]
Hi @ajith-ab, this was/is the real fix: https://github.com/ajith-ab/react-native-receive-sharing-intent/pull/3

And only after that PR is applied, should this change gone in. 

Please have a look at it.
